### PR TITLE
Win32: Various fixes for Paradox Launcher/Chrome.

### PR DESCRIPTION
**Pull request by madewokherd**

### CODE REVIEW

File: xalia/Win32/Win32Connection.cs [hunk 1]
```diff
@@ -109,7 +109,7 @@ public string GetElementName(IntPtr hwnd, int idObject=OBJID_CLIENT, int idChild
                 case OBJID_CLIENT:
                     if (idChild == CHILDID_SELF)
                         goto case OBJID_WINDOW;
-                    return $"hwnd-{hwnd.ToInt64():x}-{idChild}";
+                    return $"acc2-{hwnd.ToInt64():x}-{idChild}";
```
Comment (Conmanx360) [inline xalia/Win32/Win32Connection.cs:112]: It feels a little bit odd to me to have this be hardcoded as "acc2", I guess the assumption is that the only thing that would be passed into `GetElementName` here is an IAccessible2. Might be worth adding an argument to the function to set this, but feel free to ignore this. :)

**Review by Conmanx360 — COMMENTED** [review]
Aside from my minor nit, this looks fine to me. :)

File: xalia/Win32/Win32Connection.cs (hunk 1 above)
Comment (madewokherd) [reply xalia/Win32/Win32Connection.cs:112]: I guess instead of that, we could do away with "acc2" entirely and use hwnd-HWND-CHILDID

Comment (madewokherd) [reply xalia/Win32/Win32Connection.cs:112]: At the moment, IAccessible2 is the only way we get a childid that we know is stable (so other cases will use a different naming scheme), but if that changes then "acc2" will be inaccurate.

Comment (madewokherd) [reply xalia/Win32/Win32Connection.cs]: Removed that diff, so "hwnd-" will be used in this case now.

**Pull request merged**